content/en/blog/2019/announcing-1.3/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Announcing Istio 1.3
+subtitle: Major Update
+description: Istio 1.3 release announcement.
+publishdate: 2019-09-10
+attribution: The Istio Team
+release: 1.3.0
+---
+
+We are pleased to announce the release of Istio 1.3!
+
+{{< relnote linktonote="true" >}}
+
+The theme of Istio 1.3 is User Experience:
+
+- Improve the experience of new users adopting Istio
+- Improve the experience of users debugging problems
+- Support more applications without any additional configuration
+
+Every few releases, the Istio team delivers dramatic improvements to usability, APIs, and the overall system performance. Istio 1.3 is one such release, and the team is very excited to roll out some key updates.
+
+## Intelligent protocol detection (experimental)
+
+In past releases, you had to use a special port naming format to explicitly declare the protocol for service ports. This requirement caused problems for users that did not name their ports when they added their applications to the mesh. Starting with 1.3, the protocol for outbound traffic is automatically detected as HTTP or TCP when the ports are not named according to Istio's conventions. We will be polishing this feature in the upcoming releases with support for protocol sniffing on inbound traffic as well as identifying protocols other than HTTP.
+
```

Comment:
as much as i love this feature, i would prefer not to mention it as included in the release as it generates more confusion IMHO.  probably good to mention as a forward looking statement.

@rshriram @howardjohn

Reply:
I understand what you mean. I would lean towards keeping it though. Its pretty clear (to me) that it is partially supported and we are working on fixing it

Reply:
If we keep it, let's reword this... we are not eliminating the requirement of special port naming format for this release but are working towards that... this seems to indicate it is eliminated but it is not:

```In past releases, you had to use a special port naming format to explicitly declare the protocol for service ports.```